src/server/webkit/wkPage.ts

```diff
@@ -181,7 +181,7 @@ export class WKPage implements PageDelegate {
     if (contextOptions.userAgent)
       promises.push(session.send('Page.overrideUserAgent', { value: contextOptions.userAgent }));
     if (this._page._state.mediaType || this._page._state.colorScheme)
```

Comment:
Missing `|| this._page._state.reducedMotion`

Reply:
Good catch.